packages/nextra-theme/src/index.tsx

```diff
@@ -30,10 +28,118 @@ import {
 } from '@/components'
 import { DocumentContext, MDXLayoutNav, MDXLayoutOutline, MDXLayoutPagination, NavContext } from '@/layout'
 
+const MDXWrapper: NextraMDXContent = ({ children, toc }) => {
+  const { activePath } = useContext(NavContext)!
+  const { frontMatter, timestamp, readingTime } = useContext(DocumentContext)!
+  const { locale } = useRouter()
+  const lastUpdated = timestamp ? new Date(timestamp) : null
+
+  return (
+    <>
+      <div sx={{ pt: [null, null, null, Spacing['24px']] }}>
+        <div sx={{ display: [null, null, null, 'none'], mb: Spacing['32px'] }}>
+          <MDXLayoutNav mobile />
+        </div>
+
+        <article className="graph-docs-content" sx={mdxStyles}>
+          {activePath.length > 1 && (
+            <div className="graph-docs-current-group" sx={{ display: 'none' }}>
+              {activePath.map((item) => item.title).join(' > ')}
+            </div>
+          )}
+          {frontMatter.title ? <Heading.H1>{frontMatter.title}</Heading.H1> : null}
+          {(lastUpdated || readingTime) && (
+            <Paragraph size="14px">
```

Comment:
I noticed an issue with the `graph-ts` and `graph-client` pages: the title appears _below_ the "last updated" and "reading time" stats.

![CleanShot 2024-07-29 at 23 58 16@2x](https://github.com/user-attachments/assets/13644b92-0039-4c0e-92e9-270fd104bf97)

Reply:
fixed here https://github.com/graphprotocol/docs/pull/733

Reply:
Awesome. Looks like that PR is a draft; let me know when it's ready to review @dimaMachina 👍

Reply:
@benface ready!